hazelcast/aggregator.py

```diff
@@ -0,0 +1,334 @@
+        object_data_output.write_string(self._attribute_path)
+        object_data_output.write_long(0)
+
+    def get_class_id(self):
+        return 8
+
+
+class _FloatingPointSumAggregator(_AbstractAggregator):
+    def write_data(self, object_data_output):
+        object_data_output.write_string(self._attribute_path)
+        object_data_output.write_double(0)
+
+    def get_class_id(self):
+        return 9
+
+
+class _MaxAggregator(_AbstractAggregator):
+    def write_data(self, object_data_output):
+        object_data_output.write_string(self._attribute_path)
+        object_data_output.write_object(None)
+
+    def get_class_id(self):
+        return 14
+
+
+class _MinAggregator(_AbstractAggregator):
+    def write_data(self, object_data_output):
+        object_data_output.write_string(self._attribute_path)
+        object_data_output.write_object(None)
+
+    def get_class_id(self):
+        return 15
+
+
+class _IntegerAverageAggregator(_AbstractAggregator):
+    def write_data(self, object_data_output):
+        object_data_output.write_string(self._attribute_path)
+        object_data_output.write_long(0)
+        object_data_output.write_long(0)
+
+    def get_class_id(self):
+        return 10
+
+
+class _IntegerSumAggregator(_AbstractAggregator):
+    def write_data(self, object_data_output):
+        object_data_output.write_string(self._attribute_path)
+        object_data_output.write_long(0)
+
+    def get_class_id(self):
+        return 11
+
+
+class _LongAverageAggregator(_AbstractAggregator):
+    def write_data(self, object_data_output):
+        object_data_output.write_string(self._attribute_path)
+        object_data_output.write_long(0)
+        object_data_output.write_long(0)
+
+    def get_class_id(self):
+        return 12
+
+
+class _LongSumAggregator(_AbstractAggregator):
+    def write_data(self, object_data_output):
+        object_data_output.write_string(self._attribute_path)
+        object_data_output.write_long(0)
+
+    def get_class_id(self):
+        return 13
+
+
+def count(attribute_path=None):
+    """Creates count aggregator.
+
+    Accepts ``None`` input values and ``None`` extracted values.
+
+    Args:
+        attribute_path (str): extracts values from this path if given
+
+    Returns:
+        Aggregator[int]: an aggregator that counts the input values
+    """
+    return _CountAggregator(attribute_path)
+
+
+def double_avg(attribute_path=None):
+    """Creates double average aggregator.
+
+    Does NOT accept null input values.
+    Accepts only ``float`` input values.
+
+    Args:
+        attribute_path (str): extracts values from this path if given
+
+    Returns:
+        Aggregator[float]: an aggregator that calculates the average of
+        the input values
+    """
+    return _DoubleAverageAggregator(attribute_path)
+
+
+def double_sum(attribute_path=None):
+    """Creates double sum aggregator.
+
+    Does NOT accept null input values.
```

Comment:
There are still some `null` words left in the documentation. Can you go over all of them and replace them with 
```
``None``
```